conjur-api.gemspec

```diff
@@ -18,9 +18,9 @@ Gem::Specification.new do |gem|
 
   gem.required_ruby_version = '>= 1.9'
 
-  gem.add_dependency 'rest-client', '~> 1.7', '>= 1.7.3'
+  gem.add_dependency 'rest-client', '~> 2.0'
```

Comment:
This is great, i was in gem hell the other day with some internal project because this dep was too old. Thanks!